doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -150,3 +149,45 @@ WARNING: You should only import Dashbuilder data to a clean installation of {PRO
 . Click *Import*.
 
 endif::DM[]
+
+ifdef::PAM[]
+
+== Deploying dashboards from {CENTRAL} on Dashbuilder Runtime
+
+You can automatically deploy the dashboards from {CENTRAl} on Dashbuilder Runtime. {CENTRAl} is linked to Dashbuilder Runtime using a gradual export feature.
+
+.Prerequisites
+
+* Dashbuilder Runtime is configured on your system.
+* You have set the `dashbuilder.runtime.multi` system property to `true` in the `standalone.xml` file.
+* You have set the value of `dashbuilder.runtime.location` system property to the Dashbuilder Runtime URL. For example:
++
+[source]
+----
+<property name="dashbuilder.runtime.location" value=" http://localhost:8080"
+----
+* You have set the `dashbuilder.export.dir` system property to the shared directory where Dashbuilder Runtime reads its model. For example:
++
+[source]
+----
+<property name="dashbuilder.export.dir" value="/tmp/dashbuilder/models/"
+----
+
+
+.Procedure
+
+. Start {CENTRAL}, go to *Menu* → *Design* → *Pages*.
```

Comment:
Incomplete sentence, but more importantly, throughout the docs, we typically don't say to start BC, but instead, the fairly standard format we use most places is "In {CENTRAL}, go to ...".

Reply:
Done